### DISCUSSION TITLE
change the min. number of characters

### DISCUSSION THREAD

**Discussion by widehardomarcel** [night/betterttv#6053] [Ideas]
I love BTTV, but I HATE that you have to use at least 4 characters. For example: I wanted to add emotes like "lol" "npc" or "Pag", but because of the 4 characters limit, I had to rename them to "Lols", "NPCS" and "Pagg". And to be honest, that just looks stupid. So why don't change it? Other third-person applications like 7TV already have the feature and it just works fine (and we all know BTTV already stole features from 7TV like wide-animated emotes, the option to rename emotes,...). At the moment I think, 7TV is superior.

**Comment by dclstn** [comment]
I understand your frustration with the 4-character limit on BTTV emotes, but this restriction helps prevent unintentional conflicts with natural words, ensuring a smoother user experience. While 7TV might offer shorter emote names, BTTV aims to strike a balance between customization and minimizing potential issues.

**Comment by night** [comment]
Closing as resolved — the 4-character minimum is intentional to prevent conflicts with common words. See dclstn's comment above for context.
